source/fundamentals/crud/read-operations/geo.txt

```diff
@@ -0,0 +1,310 @@
+===================
+Search Geospatially
+===================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+On this page you will learn the how to search **geospatial data** in the
+MongoDB Java Driver.
+
+Geospatial data is data that represents a geographical location on
+the surface of the Earth. Examples of geospatial data include:
+
+- Locations of movie theaters
+- Borders of countries
+- Routes of bicycle rides
+- Dog exercise areas in New York City
+
+You should read this guide if you would like to:
+
+- Understand the different geospatial data formats in MongoDB.
+- Build geospatial queries.
```

Comment:
I still think this separates the purpose of the page into two sections, with a definition of geospatial data wedged in-between.

```suggestion
On this page you will learn the how to search **geospatial data** with the
MongoDB Java Driver, and the different geospatial data formats supported by MongoDB.

Geospatial data is data that represents a geographical location on
the surface of the Earth. Examples of geospatial data include:

- Locations of movie theaters
- Borders of countries
- Routes of bicycle rides
- Dog exercise areas in New York City
```